androidsdk/src/main/java/com/wootric/androidsdk/Wootric.java

```diff
@@ -61,23 +63,47 @@
     /**
      * It configures the SDK with required parameters.
      *
-     * @param activity Activity where the survey will be presented.
+     * @param fragmentActivity FragmentActivity where the survey will be presented.
      * @param clientId Found in API section of the Wootric's admin panel.
      * @param clientSecret Found in API section of the Wootric's admin panel.
      * @param accountToken Found in Install section of the Wootric's admin panel.
      */
     @Deprecated
-    public static Wootric init(FragmentActivity activity, String clientId, String clientSecret, String accountToken) {
+    public static Wootric init(FragmentActivity fragmentActivity, String clientId, String clientSecret, String accountToken) {
         Wootric local = singleton;
         if(local == null) {
             synchronized (Wootric.class) {
                 local = singleton;
                 if(local == null) {
-                    checkNotNull(activity, "Activity");
+                    checkNotNull(fragmentActivity, "FragmentActivity");
                     checkNotNull(clientId, "Client Id");
                     checkNotNull(clientSecret, "Client Secret");
                     checkNotNull(accountToken, "Account Token");
-                    singleton = local = new Wootric(activity, clientId, clientSecret, accountToken);
+                    singleton = local = new Wootric(fragmentActivity, clientId, clientSecret, accountToken);
+                }
+            }
+        }
+
+        return local;
+    }
+
+    /**
+     * It configures the SDK with required parameters.
+     *
+     * @param fragmentActivity FragmentActivity where the survey will be presented.
+     * @param clientId Found in API section of the Wootric's admin panel.
+     * @param accountToken Found in Install section of the Wootric's admin panel.
+     */
+    public static Wootric init(FragmentActivity fragmentActivity, String clientId, String accountToken) {
+        Wootric local = singleton;
```

Comment:
What's a singleton do?

Reply:
It prevents from creating multiple instances of the same class. So no matter how many Wootric objects they create, they will all point to the same one.